v2/api-validator/src/server/controllers/collateral/collateral-controller.ts

```diff
@@ -324,6 +325,26 @@ export class CollateralController {
   public getCurrentSettlementInstructions(): SettlementInstructions {
     const currentSettlement = this.settlementRepository.list();
 
+    currentSettlement[0].depositInstructions.forEach((instruction) => {
+      if (instruction.destinationAddress && instruction.destinationAddress.asset) {
+        instruction.destinationAddress.asset = {
+          blockchain: config.get('collateral.asset.blockchain'),
+          cryptocurrencySymbol: config.get('collateral.asset.symbol'),
+          testAsset: config.get('collateral.asset.testAsset'),
+        };
+      }
+    });
+
+    currentSettlement[0].withdrawInstructions.forEach((instruction) => {
+      if (instruction.sourceAddress && instruction.sourceAddress.asset) {
+        instruction.sourceAddress.asset = {
+          blockchain: config.get('collateral.asset.blockchain'),
+          cryptocurrencySymbol: config.get('collateral.asset.symbol'),
+          testAsset: config.get('collateral.asset.testAsset'),
+        };
+      }
+    });
```

Comment:
I thought we decided to add a new item. It isn't too critical, but, in any case, even if you overwrite, don't do it in `getCurrentSettlementInstructions`. It is a getter, why would it modify the data? Patch the data right after it is initialized.